src/test/resources/features/AcceptInterest.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Accept Interest
+    In order to accept interest
+    As a User
+    I want to accept an invite
+
+    
+Background:
+    Given There is no registered user with username "user"
+    And I'm not logged in
+    When I register a new user with username "user", email "user@sample.app" and password "password"
+    Then The response code is 201
+    And It has been created a user with username "user" and email "user@sample.app", the password is not returned
+    And I can login with username "user" and password "password"
+
+
+Scenario: Try to accept an already created interest
+    Given I can login with username "user" and password "password"
+    And There is a proposal created
+    When There already is an interest with the following details:
+      | proposalId  | 1     |
```

Comment:
Please, to avoid problems with IDs as you cannot fix them (they are generated by the database) better use the proposal title or any other attribute that you fix and know will not change. Then, retrieve the corresponding Proposal using the findByTitle in the ProposalRepository